apps/api-catalogue-e2e/src/integration/app.spec.ts

```diff
@@ -1,13 +1,8 @@
-import { getGreeting } from '../support/app.po'
-
 describe('api-catalog', () => {
   beforeEach(() => cy.visit('/'))
 
-  it('should display welcome message', () => {
-    // Custom command example, see `../support/commands.ts` file
-    cy.login('my-email@something.com', 'myPassword')
-
-    // Function helper example, see `../support/app.po.ts` file
-    getGreeting().contains('Welcome to api-catalog!')
+  it('should display homepage', () => {
+    cy.visit('/')
```

Comment:
I guess this is unnecessary as `beforeEach` calls `cy.visit('/')

Reply:
It totally is. I forgot to remove the beforeEach block. Do you mind if I fix that in a separate pr so I can skip going through the review again?